style.css

```diff
@@ -0,0 +1,586 @@
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-intro {
+  background: var(--clr-light) url('./assets/logos/logo-30anniv-2c-uai-516x470.png') no-repeat top;
+  height: 100%;
+  width: 100%;
+  background-blend-mode: overlay;
+  padding: 30px 10px;
+}
+
+.intro-greeting,
+.intro-title {
+  color: var(--clr-accent);
+}
+
+.intro-greeting {
+  font-size: 1.5rem;
+}
+
+.intro-title {
+  font-weight: var(--fw-bold);
+  font-size: var(--fs-h1);
+  line-height: normal;
+  margin: 0;
+}
+
+.intro-description {
+  border: 3px solid white;
+  background-color: var(--clr-light2);
+}
+
+.main-events {
+  background: var(--clr-dark) url('./assets/hitmar-resized.png');
+  color: var(--clr-light);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  padding: 2rem 1rem;
+  gap: 10px;
+}
+
+.events-cards {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.events-card {
+  display: grid;
+  grid-template-columns: 1fr 2fr 3fr;
+  align-items: center;
+  padding: 0.5rem;
+  background-color: rgba(120, 121, 133, 0.199);
+  gap: 0.5rem;
+}
+
+.more-info {
+  color: var(--clr-light);
+  background-color: var(--clr-accent);
+  border-radius: 0.25rem;
+  text-decoration: none;
+  text-align: center;
+  padding: 1rem;
+  width: 250px;
+  margin-top: 20px;
+}
+
+.events-icon {
+  width: 5rem;
+}
+
+.events-title,
+.featured-title {
+  margin-bottom: 0;
+}
+
+.events__card-title {
+  color: var(--clr-accent);
+  text-align: center;
+}
+
+hr {
+  width: 15%;
+  border: 1px solid var(--clr-accent);
+}
+
+.featured-artist {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  width: 100%;
+  padding: 2rem 1rem;
+}
+
+.featured-artist-container {
+  display: grid;
+  margin: 2rem 0;
+  gap: 3rem;
+}
+
+.featured-card {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.featured-image {
+  width: 8rem;
+  height: 8rem;
+}
+
+.featured-text-info {
+  display: flex;
+  flex-direction: column;
+}
+
+.featured-text-info > * {
+  margin: 2px 0;
+}
+
+.feature-card__category {
+  color: var(--clr-accent);
+  font-weight: 600;
+  font-size: 1rem;
+  font-style: italic;
+}
+
+.featured-description {
+  font-size: 0.9rem;
+}
+
+.card-hr {
+  border: 1px solid var(--clr-light);
+}
+
+.more {
+  color: var(--clr-dark);
+  border: 1px solid var(--clr-dark);
+  border-radius: 3px;
+  text-decoration: none;
+  text-align: center;
+  padding: 1rem;
+  width: 300px;
+  margin-top: 20px;
+}
+
+.more .fa-angle-down {
+  color: var(--clr-accent);
+}
+
+.main-sponsors {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  background-color: var(--clr-dark);
+  color: white;
+}
+
+.sponsors-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  list-style: none;
+  gap: 10px;
+  padding: 0 20px;
+}
+
+.sponsors-item img {
+  width: 8rem;
+  height: 5rem;
+  mix-blend-mode: overlay;
+}
```

Comment:
- Hey, let's add a filter effect on these images to be close to the original design. you can add this  
` filter: grayscale(100%) contrast(30%);`. You can play around with the contrast to adjust it.